traits.yml

```diff
@@ -45,30 +58,123 @@ traits:
         properties:
           name:
             type: string
-            description: |
+            description: >
               The humanized name of entity itself, not including any
               hierarchy or classification.
 
+
               For example:
               - `"Cuttlefish v1"` - for a version of an asset
               - `"seq003"` - for a sequence in a hierarchy
 
           qualifiedName:
             type: string
-            description: |
-              An unambiguous, humanised name for the entity.
+            description: >
+              An unambiguous, humanized name for the entity.
+
 
               The display name may want to consider the host, and any other
               relevant Context information to form a display name for an
               entity that can uniquely identify the entity in that context.
 
+
               For example:
               - `"dive / build / cuttlefish / model / v1"` for a version
                   of an asset in an 'open recent' menu.
               - `"Sequence 003 [ Dive / Episode 1 ]"` for a sequence in
-                 an hierarchy as a window title.
+                 a hierarchy as a window title.
+  lifecycle:
+    description: >
+      Traits that are concerned with describing aspects of the lifecycle
+      of an entity, such as versioning.
+    members:
+      Version:
+        description: >
+          Describes a specific version of an entity.
+
+
+          We define a version as a revision or iteration of what is
+          otherwise the same logical entity. Versions are created when
+          new data is published to an existing entity reference of the
+          same trait set.
```

Comment:
Hows [this](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/51/commits/97908a151929f530c59257e97730dc6e4c2ca53c)?

Reply:
[Taking over for Elliot who is away]  Better.  Though I wonder if we should be more hand-wavy? E.g. 

```
          A version is defined as a revision or iteration of what is
          otherwise the same logical entity. For example,
          versions may be created when new data is published 
          to an existing entity.
```

Reply:
Is there another way though? Thats how it's done in the API.  We really need to be concrete somewhere, so people know how this all works. We're deliberately vague in the core API about certain things, to support domain specific application. Without some clarity in MC, then i think we risk no one really knowing what to do. MC should be the "rules for VFX (and friends)".

Reply:
OK that's fine.  On re-reading, there isn't much difference between my suggestion and your updated text anyway - both imply that versions may or may not be created, it's up to the manager.  I'm happy with your text.